File: (5 kyu) Count IP Addresses/(5 kyu) Count IP Addresses.js
const ipsBetween = (start, end) => {
  const calc = (n, m = 1) => (end.split`.`[n] - start.split`.`[n]) * m;

  return calc(0, 256 * 256 * 256) + calc(1, 256 * 256) + calc(2, 256) + calc(3);
};

File: 5kyu/(5 kyu) Count IP Addresses/(5 kyu) Count IP Addresses.py
def ips_between(start, end):
    calc = lambda n, m: (int(end.split(".")[n]) - int(start.split(".")[n])) * m
    return calc(0, 256 * 256 * 256) + calc(1, 256 * 256) + calc(2, 256) + calc(3, 1)
